src/stdlib_hashmap_wrappers.f90

```diff
@@ -277,6 +279,21 @@ subroutine get_int8_key( key, value )
     end subroutine get_int8_key
 
 
+    subroutine get_int32_key( key, value )
+!! Version: Experimental
+!!
+!! Gets the contents of the key as an INTEGER(INT32) vector
+!! Arguments:
+!!     key   - the input key
+!!     value - the contents of key mapped to an INTEGER(INT32) vector
+        type(key_type), intent(in)              :: key
+        integer(int32), allocatable, intent(out) :: value(:)
+        
+        value = transfer( key % value, value )
```

Comment:
Here I wonder whether it matters that `value` is unallocated in the RHS call to `transfer` (since it will be deallocated on input if allocated)?

Reply:
I've asked this question at fortran discourse [here](https://fortran-lang.discourse.group/t/using-transfer-with-unallocated-mold/7867).

Reply:
Thanks for asking.  Intuitively it seems like it shouldn't matter since its only pulling type info, not size.  But I also did a quick search and didn't come up with anything.   We could hardcode it to [0_int8], but I don't like that quite as much.